http-client/Network/HTTP/Client/Manager.hs

```diff
@@ -20,22 +20,23 @@ module Network.HTTP.Client.Manager
     , useProxySecureWithoutConnect
     ) where
 
-import qualified Data.ByteString.Char8 as S8
+import qualified Data.ByteString.Char8          as S8
 
-import Data.Text (Text)
+import           Data.Text                      (Text)
 
-import Control.Monad (unless)
-import Control.Exception (throwIO, fromException, IOException, Exception (..), handle)
+import           Control.Exception              (Exception (..), IOException,
+                                                 fromException, handle, throwIO)
+import           Control.Monad                  (unless)
 
-import qualified Network.Socket as NS
+import qualified Network.Socket                 as NS
 
-import Network.HTTP.Types (status200)
-import Network.HTTP.Client.Types
-import Network.HTTP.Client.Connection
-import Network.HTTP.Client.Headers (parseStatusHeaders)
-import Network.HTTP.Proxy
-import Data.KeyedPool
-import Data.Maybe (isJust)
+import           Data.KeyedPool
```

Comment:
I'd prefer to receive a PR without unnecessary code formatting changes. Can you resubmit with just the relevant changes?

Reply:
Good! Please expect changes soon.

Reply:
Formatting changes undone, commits squashed. Now the diff contains only relevant changes.